### ISSUE TITLE
Support for previous methods of import in SASS

### ISSUE THREAD

**Issue by JacobWeinbren** [glromeo/esbuild-sass-plugin#23]
Hi - thanks for making this plugin, it is really useful.

My issue is SASS dependencies often use the now (depreciated) ~ (tilda) to import from node modules.

This produces the following error on the plugin.

```
error: Cannot find module 'mathsass/dist/math' from <repo>
  ╷
1 │ @import '~mathsass/dist/math';
  │         ^^^^^^^^^^^^^^^^^^^^^
  ╵
```

For record, this is is produced the following line of code `@import "~@aotearoan/sass-farbig";` in `theme.scss` using the latest versions of esbuild and esbuild-sass-plugin.

I have also tried the `includePaths` option, producing the same error
```
[
    sassPlugin({
        type: "style",
        includePaths: [path.resolve(__dirname, "node_modules")],
    }),
],
```

Webpack, Rollup are other compilers all have support for this edge case. Do you have any advice on what to do if dependencies include this syntax and I am using esbuild and this sass plugin?

Jacob

**Comment by glromeo** [comment]
@JacobWeinbren the error you see is due to the importer within the plugin not being able to resolve mathsass/dist/math because it is a fragment (the file is _math.scss which is ignored by the resolve.sync method used by the importer which is really meant for node rather than sass and its convoluted resolution mechanism). I have a fix for that and you can find it in v1.4.13

**Comment by JacobWeinbren** [comment]
@glromeo Thank you it works now.
